### PR TITLE
feat: combine outlook and wxp extensions on Toolkit

### PR DESCRIPTION
**Pull request by YueLi-MSFT**
### Background:

- [PM Spec](https://microsoft.sharepoint.com/:w:/t/OfficePlatform/EQTzlom9CSFGgAOIro3-sDsBTsmVkXxz6IWCUoJVLp6L9w?e=vW0Dla)

- [ADO Link](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/27130363)

### Description
This is the implement of the above PM Spec section create a WXPO add-in through VS Code with guidance.

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -287,6 +287,11 @@
   "core.createProjectQuestion.projectType.outlookAddin.detail": "Customize the ribbon and Task Pane with your web content",
   "core.createProjectQuestion.projectType.outlookAddin.label": "Outlook Add-in",
   "core.createProjectQuestion.projectType.outlookAddin.title": "App Features Using an Outlook Add-in",
+  "core.createProjectQuestion.projectType.officeAddin.detail": "Extend Office applications and interact with content in Office documents and Outlook mails",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:290]: Proposed: Extend Office apps to interact with content in Office documents and Outlook mails

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -507,6 +512,7 @@
   "core.common.ReceiveApiResponse": "Received API response: %s.",
   "core.importAddin.label": "Import an Existing Outlook Add-ins",
   "core.importAddin.detail": "Upgrade an Add-ins project to the latest app manifest and project structure",
+  "core.importOfficeAddin.label": "Import an Existing Office Add-ins",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:515]: Article not required in label.
Proposed: Import Existing Office Add-ins

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (YueLi-MSFT) [reply packages/fx-core/resource/package.nls.json:289]: Thanks！I will change it.

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (YueLi-MSFT) [reply packages/fx-core/resource/package.nls.json:514]: Same as above. Thanks Supkasar!

**Review by jayzhang — APPROVED** [review]

**Review by supkasar — APPROVED** [review]

**Pull request merged**